### ISSUE TITLE
Custom module that is persistent

### ISSUE THREAD

**Issue by AlessioDP** [Alexays/Waybar#174]
I'm trying to figure out how custom modules works.

What should I do if I want make a custom script that returns EXAMPLE and that message should stay fixed into the bar?

Example of `test-command`:
```bash
#!/bin/sh
printf "message"
```

Example of custom module:
```
"custom/test": {
  "format": "{}",
  "exec": "test-command",
  "interval": 10 // Do the command every 10 seconds
}
```

For now, if I don't put a sleep at the end of the bash it won't work (I cannot see the module, and I receive the message `test just stopped, is it endless?`).

**Comment by Alexays** [comment]
Now fixed with 0c9edb0c4b79be195a090df8b2e78ca3f4f2fbc3
When interval is omited and the script exit normally, i don't hide the module

**Comment by coredotbin** [comment]
Can't this also be done with
`"format": "Static text"`
